internal/ui/confirmation/confirmation.go

```diff
@@ -51,7 +52,13 @@ func WithStylePrefix(prefix string) Option {
 // WithOption adds an option to the confirmation dialog
 func WithOption(label string, cmd tea.Cmd, keyBinding key.Binding) Option {
 	return func(m *Model) {
-		m.options = append(m.options, option{label, cmd, keyBinding})
+		m.options = append(m.options, option{label, cmd, keyBinding, cmd})
+	}
+}
+
+func WithAltOption(label string, cmd tea.Cmd, altCmd tea.Cmd, keyBinding key.Binding) Option {
```

Comment:
This is fine for now, but I will probably revisit this because it is not configurable and also doesn't work on my Mac, which is fine as it is not taking anything away functionality wise.